pkgs/by-name/ha/harlequin/package.nix

```diff
@@ -29,6 +27,21 @@ python3Packages.buildPythonApplication rec {
   nativeBuildInputs = [ glibcLocales ];
 
   dependencies =
+    let
+      # Temporarily use textual 0.82 until harlequin is updated for 0.86
+      # https://github.com/NixOS/nixpkgs/issues/364867
+      textual_0_82 = (
+        python3Packages.textual.overrideAttrs {
+          version = "0.82.0";
+          src = fetchFromGitHub {
+            owner = "Textualize";
+            repo = "textual";
+            rev = "refs/tags/v${version}";
+            hash = "sha256-belpoXQ+CkTchK+FjI/Ur8v4cNgzX39xLdNfPCwaU6E=";
+          };
+        }
+      );
+    in
```

Comment:
The standard from what I've seen is to declare these at the top right before the derivation, so basically in a `let..in` directly above `python3Packages.buildPythonApplication rec {`

Reply:
I see. Just updated it. Thanks. :+1: